### PR TITLE
User Profile Screen

### PR DESCRIPTION
**Pull request by chaitanyas16**
User Profile screen changes as per Issue 241

### CODE REVIEW

**Comment by Code4SocialGood** [comment]
Please resolve conflicts

File: src/app/user/profile/user-profile.component.ts [hunk 1]
```diff
@@ -1,22 +1,31 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { Validators, FormGroup, FormControl } from '@angular/forms';
+import { FormConstantsService } from '../../_services/form-constants.service';
+import { environment } from '../../../environments/environment';
+import {Http} from '@angular/http';
+
+
+const skillUrl = `${environment.backend_url}/api/skills`;
```
Comment (Code4SocialGood) [inline src/app/user/profile/user-profile.component.ts]: Retrieve skills from skill.service.ts

File: src/app/user/profile/user-profile.component.ts [hunk 2]
```diff
@@ -27,7 +36,38 @@ export class UserProfileComponent {
...
+
+  constructor(private fc: FormConstantsService, private http: Http) {
+  }
+
+  private getFormConstants(): void {
+
+    this.countries = this.fc.getCountries();
+  }
+
+  getSkills() {
```
Comment (russon77) [inline src/app/user/profile/user-profile.component.ts]: any http requests should be in a service

File: src/app/user/profile/user-profile.component.ts (hunk 1 above)
Comment (chaitanyas16) [inline src/app/user/profile/user-profile.component.ts]: I will fix this.

File: src/app/user/profile/user-profile.component.ts (hunk 2 above)
Comment (chaitanyas16) [inline src/app/user/profile/user-profile.component.ts]: I will include skill.service.ts here and remove the http request.

**Comment by chaitanyas16** [comment]
Closing the pull request as there were some issues with the version of the code.

**Pull request closed without merging**